default/scripting/buildings/GATEWAY_VOID.focs.txt

```diff
@@ -34,7 +34,7 @@ BuildingType
                 InSystem id = Source.SystemID
             ]
             activation = Turn low = Source.CreationTurn + 1
-            priority = [[EARLY_POPULATION_OVERRIDE_PRIORITY]]
+            priority = [[EARLY_POPULATION_OVERRIDE_PRIORITY]]  // implies overriding target pop effects
```

Comment:
this comment is unclear... or why is it not EARLY_TARGET_POPULATION_OVERRIDE_PRIOIRITY ...?

Reply:
Would an extra "also" help?
```// also implies overriding target pop effects```

The effect should override both target pop and population. Not sure how to word it clearer.

That being said, it seems this new priority means that supply isn't overridden as most supply effects will/may happen later than population effects. Maybe we need to split the effect with the supply effect happening at a later priority so that different supply effects are correctly overridden?

Reply:
Seems to me that anything more here is really just about pre-existing issues with BLD_GATEWAY_VOID, really independent of the target population effects.  A new issue could be opened up regarding possibly splitting the effects of BLD_GATEWAY_VOID, but I don't think we should be doing more with it here beyond the possible clarification to the comment that Morlic has proposed.

Reply:
Maybe this?
// overrides both Population and TargetPopulation